.github/workflows/release-e2e.yaml

```diff
@@ -0,0 +1,49 @@
+---
+name: release-e2e
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  release-e2e-kind:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '~1.18'
+
+    - name: start_kind_cluster
+      run: |
+        make kind-cluster
+
+    - name: install_kubectl
+      run: |
+        curl -LO "https://dl.k8s.io/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl"
+        sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+
+    - name: Set the release related variables
+      run: |
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            IMAGE_TAG=${GITHUB_REF#refs/tags/} >> $GITHUB_ENV
+          fi
+
+    - name: install_cert_manager
+      run: |
+        make cert-mgr
+
+    - name: release_install
+      run: |
+        kubectl apply -f https://github.com/operator-framework/rukpak/releases/download/${{ IMAGE_TAG }}/rukpak.yaml
```

Comment:
Same comment as before: I think we need to use {{ env.IMAGE_TAG }} here if we're indexing into the github environment context.